docs/configuration.rst

```diff
@@ -139,7 +139,14 @@ multiple choices of matching are possible.
 We could match ``seg1/seg2`` with the first token, and ``seg3`` with the second token,
 we could also match ``seg1`` with the first token, and ``seg2/seg3`` with the second token, etc.
 
-The variables and tokens are useful to know the type of any segments that doesn't have a fixed name.
+In addition of using the name of the segment it is possible to use instead a key found in `Magpie`_ resource schema
```

Comment:
Yes it would use the `name` key to generate the regex, but would use the field key to generate the `resource nametype path` . 

Default Case : Only `name` is used in the config : the key 'name' match the name or the segment with a regex generated from the resource name.
Case 2: the `name` is used in the config with a `field:resource_name` (Same as default case)
Case 3: the `name` is used in the config with a `field` different from `resource_name` let's say `resource_display_name`. It will generate the `resource nametype` using the `resource_display_name` ( if the resource_display_name is the same as the resource_name we are back at the default case) In both cases ,the regex is always generated using the `name` key.
Case 4: field + regex in the config (behaviour explained previously)
Case 5: Name + regex (not allowed it's defined in the `config.py`)